app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -347,7 +349,7 @@ public Integer getSessionId(GeckoSession aSession) {
         return null;
     }
 
-    public String gerUriFromSession(GeckoSession aSession) {
+    public String getUriFromSession(GeckoSession aSession) {
```

Comment:
there was a typo here: `ger` -> `get`

Reply:
actually, I could remove this method completely - since I removed its only [use below](https://github.com/MozillaReality/FirefoxReality/pull/402/files#r212514020). for another time… 👍